main.nf

```diff
@@ -73,24 +72,24 @@ process manta {
     output:
         file 'manta.vcf' into manta_vcf
 
-    publishDir params.outdir, mode: 'copy'
-
-    module 'bioinfo-tools'
-    module "$params.modules.manta"
+    publishDir params.outdir, mode: 'copy', saveAs: { "$params.prefix$it" }
 
     errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
-    time { params.long_job * 2**(task.attempt-1) }
+    time { params.runtime.caller * 2**(task.attempt-1) }
     maxRetries 3
     queue 'core'
     cpus 4
 
+    module 'bioinfo-tools'
+    module "$params.modules.manta"
+
     when: 'manta' in workflowSteps
 
     script:
     """
     configManta.py --normalBam bamfile --referenceFasta $params.ref_fasta --runDir testRun
     cd testRun
```

Comment:
Minor: Instead of `cd dir; do_stuff; cd ..`, do `( cd dir && do_stuff )`. That way you get

1. No `cd` here and back (it's local to the sub-shell).
2. No weirdness if the directory isn't accessible (`cd` fails).